src/index.html

```diff
@@ -11,7 +11,122 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <main>
+      <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+        <fieldset>
+          <legend>Personal information</legend>
+          <label class="label" for="surname">
+            Surname:
+            <input
+            type="text"
+            autocomplete="off"
+            name="surname"
+            >
```

Comment:
check all indents

make the same as in the checklist
<img width="957" alt="image" src="https://user-images.githubusercontent.com/102411130/228765025-acfb66af-6937-4ee5-b034-6f33aac211aa.png">